README.md

```diff
@@ -53,6 +53,101 @@ Add this to your project's `pom.xml` file:
 ```
 
 
+## Getting Started
+
+### Add your Android Platform
+To initialize your SDK and start interacting with Appwrite services, you need to add a new Android platform to your project. To add a new platform, go to your Appwrite console, choose the project you created in the step before, and click the 'Add Platform' button.
+
+From the options, choose to add a new **Android** platform and add your app credentials.
+
+Add your app <u>name</u> and <u>package name</u>. Your package name is generally the applicationId in your app-level `build.gradle` file. By registering a new platform, you are allowing your app to communicate with the Appwrite API.
+
+### OAuth
+In order to capture the Appwrite OAuth callback url, the following activity needs to be added to your [AndroidManifest.xml](). Be sure to replace the **[PROJECT_ID]** string with your actual Appwrite project ID. You can find your Appwrite project ID in your project settings screen in the console.
+
+```xml
+<manifest>
+    <application>
+        <activity android:name="io.appwrite.views.CallbackActivity" >
+            <intent-filter android:label="android_web_auth">
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <data android:scheme="appwrite-callback-[PROJECT_ID]" />
+            </intent-filter>
+        </activity>
+    </application>
+</manifest>
+```
+
+### Init your SDK
+
+<p>Initialize your SDK code with your project ID, which can be found in your project settings page.
```

Comment:
```suggestion
<p>Initialize your SDK code with your Appwrite server API endpoint and project ID, which can be found in your project settings page.
```

We should probably fix this in other tutorials as well.

Reply:
Done :+1: